app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -281,16 +281,15 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
     }
   }
 
-  override fun populateList(
-      prescribedDrugs: List<PrescribedDrug>,
-      medicalHistory: MedicalHistory
-  ) {
+  override fun populateList(prescribedDrugs: List<PrescribedDrug>) {
```

Comment:
Rename to `populateDrugs()`

Reply:
I intentionally did not rename this because this method will be removed in #1038.

Reply:
Yea I got that. But think it is always a good idea to ensure every PR is at least complete in itself for other people who are working on the code.
It is okay to not change this for now.